packages/compass-components/src/components/generative-ai/generative-ai-input.tsx

```diff
@@ -110,14 +114,10 @@ const buttonResetStyles = css({
   cursor: 'pointer',
 });
 
-const closeAIButtonStyles = css(
-  buttonResetStyles,
-  {
-    padding: spacing[1],
-    display: 'inline-flex',
-  },
-  focusRing
-);
+const closeAIButtonStyles = css(buttonResetStyles, focusRing, {
+  padding: spacing[1],
+  position: 'absolute',
```

Comment:
If you're changing controls to absolute positioning, you probably want to adjust the input size too, otherwise text can go below buttons:

![image](https://github.com/mongodb-js/compass/assets/5036933/19fbf7c6-d8ce-4dbb-9106-4099fa739313)

Reply:
Good catch, updated to have a restricted width:
<img width="244" alt="Screenshot 2023-08-18 at 5 05 57 PM" src="https://github.com/mongodb-js/compass/assets/1791149/707626d1-5189-4140-93c8-b98cf98a01b5">